control_plane/src/endpoint.rs

```diff
@@ -454,10 +454,10 @@ impl Endpoint {
                         .env
                         .safekeepers
                         .iter()
-                        .map(|sk| format!("localhost:{}", sk.get_compute_port()))
+                        .map(|sk| format!("host=localhost port={}", sk.get_compute_port()))
                         .collect::<Vec<String>>()
                         .join(",");
-                    conf.append("neon.safekeepers", &safekeepers);
+                    conf.append("neon.safekeeper_connstrings", &safekeepers);
```

Comment:
Some compat tests are broken because I think it runs old computes (without `neon.safekeeper_connstrings`) with new control plane.
This kind of changes are usually done with several steps (several PRs):
1. First, add new option to compute, but do not use it in CP (or use under a feature flag).
2. Wait for the new option to reach the release
3. Change the option in CP
4. Delete old option

It's quite complicated and not fast, that's why I liked the idea of a separate `extra_conn_options`

Reply:
Do I understand correctly that we need to restart all computes to switch this options in control plane in production?